README.md

```diff
@@ -21,15 +21,193 @@ The `LSP6 ExcuteRelayCall` prefix is used instead of the standard Ethereum trans
 ## Install
 
 ```bash
-npm install @lukso/lsp6-signer.js
+npm install @lukso/eip191-signer.js
 ```
 
 ## Setup
```

Comment:
Better `Usage`
And just make  short examples in the usage section like:

```
import ()...

new class...

signEtheuem.....

recover()...
// result

signValidator...

recover()
// result
```

Lets not repeat the docs here at length. :)